types/three/examples/jsm/transpiler/TSLEncoder.d.ts

```diff
@@ -1,7 +1,30 @@
-import { Program } from "./AST.js";
+import { Program, Statement } from "./AST.js";
 
 export default class TSLEncoder {
     constructor();
 
+    tab: string;
+    imports: Set<string>;
+    global: Set<string>;
+    overloadings: Map<string, string>;
+    iife: boolean;
+    uniqueNames: boolean;
+    reference: boolean;
+
+    addImport(name: string): void;
+    emitUniform(node: Statement): string;
+    emitExpression(node: Statement): string;
+    emitBody(body: Statement): string;
```

Comment:
This one is an array, right?

```suggestion
    emitBody(body: Statement[]): string;
```